src/components/Contacts/ContactDetails/ContactDetailsHeader/ContactDetailsHeader.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement } from 'react';
+
+import { GqlMockedProvider } from '../../../../../__tests__/util/graphqlMocking';
+
+import { ContactDetailsHeader } from './ContactDetailsHeader';
+import { GetContactDetailsHeaderQuery } from './ContactDetailsHeader.generated';
+
+export default {
+  title: 'Contacts/ContactDetails/Header',
+};
+
+const accountListId = 'abc';
+const contactId = 'contact-1';
+
+export const Default = (): ReactElement => {
```

Comment:
Should we have a storybook for a loading state?

Reply:
In a previous PR, I found out how to mock a loading state using the `MockedProvider`, but I haven't found a good way to do it with `GQLMockedProvider` yet.  I need some way to keep the query from finishing in order for Storybook to keep the loading state visible.

Reply:
Daniel and I had a discussion earlier today and we agreed we could not find a good way to mock the loading state using ```GQLMockProvider``` and instead decided on using ```MockProvider``` and a delay to mock loading states. Here how I handled it in the NewsletterMenu story: https://github.com/CruGlobal/mpdx-react/pull/51/files#diff-8f40b0ad90892d047b8f9edf7b387e7ac1c7af607e31b1572f52b43b3316e2adR37